README.markdown

```diff
@@ -65,3 +78,17 @@ So most likely you want to have something like
 `bibgetter pull --file article.aux --local bibliography.bib`
 
 in your toolchain.
+
+### Adding aliases
+
+* `bibgetter alias` adds aliases for bibliography items to the central file
+
+You can add an alias to an entry by running
+
+`bibgetter alias rouquier-dimension 'https://doi.org/10.1017/is008004024jkt010'
+
+Then you can use `bibgetter get rouquier-dimension` to get the bibtex record for
+this paper. The paper will be added to the central file if it is not already there.
+
+Run `bibgetter alias` to print all defined aliases. Advanced operations (editing,
```

Comment:
When I do this on my file, it only prints this for 2 entries, but there are way more entries with an `ids` field set. Is there something off with the implementation?